### DISCUSSION TITLE
How to open port 8444 on TP Link Archer C7 ?

### DISCUSSION THREAD

**Discussion by RollingRoland** [Chia-Network/chia-blockchain#3170] [General]
Does anyone have any experience opening port 8444 on a TP Link Archer C7 router ? I added a rule under the option "Port Triggering". That is the closest thing I could find in the router menu. There seems to be no other straight up option to open a port. When I use the checking sites the port 8444 still seems to be closed. So I am not sure if I too dumb to properly set it or if my ISP uses Carrier Grade NAT to put multiple customers behind the same IP address.

My GUI has not been syncing in 24 hours. So any help would be much appreciated.

**Comment by dorkmo** [comment]
take some screen shots of the router's menu and maybe we can walk you through it.

**Comment by RollingRoland** [comment]
![ArcherC7PortTriggering](https://user-images.githubusercontent.com/83419258/116623677-33323f00-a979-11eb-88de-e52947023cce.jpg)

**Comment by RollingRoland** [comment]
This is the edit for "Port Triggering"
![AC7PTedit](https://user-images.githubusercontent.com/83419258/116624767-1bf45100-a97b-11eb-9048-e69c7f682186.jpg)
I also set a Forward Rule:
![ArcherC7ForwardingRule](https://user-images.githubusercontent.com/83419258/116624809-33cbd500-a97b-11eb-838b-6fd4bf58a3ed.jpg)
This is the edit for the "Forward Rule:
![AC7FRedit](https://user-images.githubusercontent.com/83419258/116624869-49d99580-a97b-11eb-9ed1-ea14047d0b95.jpg)

**Comment by dorkmo** [comment]
on that last one, put 8444 in the internal port also i think. you can probably delete the port "triggering" rules

(oh says you can leave blank)

**Comment by dorkmo** [comment]
you might set up a DHCP reservation for your node computer also to make sure it's network ip address doesnt change

**Comment by dorkmo** [comment]
What does the "advanced routing" tab show?

**Comment by RollingRoland** [comment]
Advanced Routing edit:
![ARedit](https://user-images.githubusercontent.com/83419258/116626423-c53c4680-a97d-11eb-81ee-af7f283818a8.jpg)
DHCP settings:
![DHCPsettings](https://user-images.githubusercontent.com/83419258/116626483-e00ebb00-a97d-11eb-876e-dd65ec5fc71b.jpg)
DHCP Address Reservation:
![DHCPAddressRes](https://user-images.githubusercontent.com/83419258/116626641-2fed8200-a97e-11eb-82d9-e01e1e24593e.jpg)

**Comment by dorkmo** [comment]
click on the DHCP Clients List, you should see all the connected computers, just look for your node. if there is an "add reservation" button that'd be nice. if not copy down the MAC address and create a reservation on that page

**Comment by dorkmo** [comment]
you could add the google DNS servers to there too https://developers.google.com/speed/public-dns/docs/using

**Comment by dorkmo** [comment]
https://www.comparitech.com/net-admin/port-forwarding-port-triggering-differences/

https://en.wikipedia.org/wiki/Port_triggering
"It is similar to port forwarding in that it enables incoming traffic to be forwarded to a specific internal host machine, although the forwarded port is not open permanently and the target internal host machine is chosen dynamically"

sounds like you definately want port forwarding, and not port triggering

**Comment by sargonas** [comment]
Tagging this issue for migration to a discussion. Apologies in advance, but you will be best served by moving this over there and out of the issue tracker, where the community can better assist you. (Ideally the issue tracker should be used to focus on bugs and development problems that need to be addressed, with general user support handled in Discussions and Keybase)

^MigrateToDiscussion
